src/course-team/info-modal/InfoModal.jsx

```diff
@@ -34,7 +34,7 @@ const InfoModal = ({
     <AlertModal
       title={title}
       variant={variant}
-      isOpen={isOpen}
+      isOpen={modalType === MODAL_TYPES.error && !errorEmail ? false : isOpen}
```

Comment:
```suggestion
      isOpen={isOpen && !(modalType === MODAL_TYPES.error && !errorEmail)
}
```

and please move it to `const`